pip-requirements.txt

```diff
@@ -1,3 +1,4 @@
 -e git+https://github.com/asl2/PyZ3950.git#egg=py3250
 ply==3.4
-boto
\ No newline at end of file
+boto
```

Comment:
Why boto?

Reply:
Is for [getting data from Amazon S3](https://github.com/datopian/ckanext-geodatagov/blob/master/ckanext/geodatagov/commands.py#L1328)

Reply:
Yes, but why are you adding it? I don't see any other references to boto or S3 in your diff. Is the dependency missing?

Reply:
I think I missed this line. Was added [here](https://github.com/datopian/ckanext-geodatagov/commit/f0afe1d6b102ad398df504676dfb160a626791dc). The requirement and the command that uses it were added at the same time.